ui/analyse/src/ctrl.ts

```diff
@@ -254,9 +254,11 @@ export default class AnalyseCtrl {
   flip = () => {
     this.flipped = !this.flipped;
     this.study?.onFlip();
-    this.chessground.set({
-      orientation: this.bottomColor(),
-    });
+    if (this.chessground) {
```

Comment:
Wouldn't just `this.chessground?.set...` work? On the other side, should the code continue if chessground is not available?

Reply:
Rest of code doesn't trigger the error, so meh; but good point regarding @ '?.'